lib/pbench/server/database/models/datasets.py

```diff
@@ -674,6 +674,9 @@ class Metadata(Database.Base):
     # UNPACKED records the size of the unpacked directory tree.
     SERVER_UNPACKED = "server.unpacked-size"
 
+    # UNPACK performance: {"best": <seconds>, "worst": <seconds>, "count": <n>}
+    SERVER_UNPACK_PERF = "server.unpack-perf"
```

Comment:
I think we're using `"max"`/`"min"` rather than `"best"`/`"worst"`.

Reply:
That's what I get for writing the "documentation" before I actually implemented any of it. (Sigh.)

Reply:
Talk to your guy who didn't follow the spec!